torchdata/dataloader2/reading_service.py

```diff
@@ -366,50 +371,62 @@ def finalize(self) -> None:
         self._worker_processes = []
         self._dispatch_process = None
 
-    def _pause(self):
+    def _pause(
+        self, pause_fn: Optional[Callable[[DataPipe], DataPipe]] = None
+    ) -> Optional[Callable[[DataPipe], DataPipe]]:
```

Comment:
Given `DataLoader2` doesn't invoke these methods with `pause_fn`, it is not obvious to users how it is intended to be used. Perhaps we should add some documentations

Reply:
Sorry, what do you mean? `DataLoader2` does invoke the returned `pause_fn`.

Reply:
I mean this line in `DataLoader2` never pass in a `pause_fn`.

https://github.com/pytorch/data/blob/c8a52da300fa0d29b8603aeed475621897d794b3/torchdata/dataloader2/dataloader2.py#L383

Reply:
Yeah. In theory, users should not realize a `pause_fn` can be provided. This is should be tied to developers who want to develop their own ReadingService.